### PR TITLE
Added 7 New Sites

### PR DESCRIPTION
**Pull request by ef1500**
Added the following sites:
Aminoapps
Artists & Clients
Roleplay.me
Cytoid
Carrd.co
Trello
Skeb

### CODE REVIEW

File: web_accounts_list.json [hunk 1]
```diff
@@ -132,6 +154,17 @@
...
         "valid" : true
       },
+      {
+        "name" : "carrd.co",
+        "check_uri" : "https://{account}.carrd.co",
+        "account_existence_code" : "200",
+        "account_existence_string" : "( Made with Carrd )",
+        "account_missing_code" : "404",
+        "account_missing_string" : "Sorry, the requested page could not be found.",
+        "known_accounts" : ["miltia"],
```
Comment (WebBreacher) [inline web_accounts_list.json:164]: Can you please add another `known_accounts` entry here? Always like having at least 2.

File: web_accounts_list.json [hunk 2]
```diff
@@ -220,6 +253,17 @@
         "category" : "images",
         "valid" : true
       },
+      {
+        "name" : "Cytoid",
```
Comment (WebBreacher) [inline web_accounts_list.json:257]: This site does not work for me. Gives me a 404 for this known_user `Cytoid --> Bad detection code and string. Received Code: 404; Expected Code: 200.`

File: web_accounts_list.json [hunk 3]
```diff
@@ -2043,6 +2087,17 @@
...
         "valid" : true
       },
+      {
+        "name" : "Roleplay.me",
+        "check_uri" : "https://www.roleplay.me/{account}",
+        "account_existence_code" : "200",
+        "account_existence_string" : "Last Visited:",
+        "account_missing_code" : "404",
+        "account_missing_string" : "The link you followed may be broken or the page may have been removed.",
+        "known_accounts" : ["adorealoi"],
```
Comment (WebBreacher) [inline web_accounts_list.json:2097]: Please add a second account to the `known_accounts`

File: web_accounts_list.json [hunk 4]
```diff
@@ -2087,6 +2142,17 @@
...
         "valid" : true
       },
+      {
+        "name" : "skeb",
+        "check_uri" : "https://skeb.jp/@{account}",
+        "account_existence_code" : "200",
+        "account_existence_string" : "Sent Requests",
+        "account_missing_code" : "404",
+        "account_missing_string" : "Not Found",
+        "known_accounts" : ["eipuru_"],
```
Comment (WebBreacher) [inline web_accounts_list.json:2152]: Please add a second account for the `known_accounts`

File: web_accounts_list.json [hunk 5]
```diff
@@ -2318,6 +2384,18 @@
...
       },
+      {
+        "name" : "Trello",
+        "check_uri" : "https://trello.com/1/Members/{account}?fields=activityBlocked%2CavatarUrl%2Cbio%2CbioData%2Cconfirmed%2CfullName%2CidEnterprise%2CidMemberReferrer%2Cinitials%2CmemberType%2CnonPublic%2Cproducts%2Curl%2Cusername",
+        "pretty_uri" : "https://trello.com/{account}",
+        "account_existence_code" : "200",
+        "account_existence_string" : "avatarUrl",
+        "account_missing_code" : "404",
+        "account_missing_string" : "Oh no! 404!",
+        "known_accounts" : ["naranjasan"],
```
Comment (WebBreacher) [inline web_accounts_list.json:2395]: Please add a second account for the `known_accounts`

**Review by WebBreacher — CHANGES_REQUESTED** [review]

File: web_accounts_list.json (hunk 2 above)
Comment (ef1500) [reply web_accounts_list.json:257]: This was caused due to a silly typo on my end. That's my bad for not realizing it

File: web_accounts_list.json (hunk 4 above)
Comment (ef1500) [reply web_accounts_list.json]: I also made the checker go through the API now, so it should be faster now!

**Comment by WebBreacher** [comment]
Thanks for the submission and the changes!

**Pull request merged**